src/main.js

```diff
@@ -0,0 +1,14 @@
+import FiltersView from './view/filters-view';
+import TripInfoPresenter from './presenter/trip-info-presenter';
+import BoardPresenter from './presenter/board-presenter';
+import { render } from './render.js';
+
+const mainElement = document.querySelector('.trip-main');
+const tripControlsFiltersElement = mainElement.querySelector('.trip-controls__filters');
```

Comment:
Кажется что можно укоротить немного нейминг
```suggestion
const tripFiltersElement = mainElement.querySelector('.trip-controls__filters');
```